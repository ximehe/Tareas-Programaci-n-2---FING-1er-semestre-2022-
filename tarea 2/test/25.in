# implementar cadenaDeColCadenas
cadenaDeColCadenas 0
cadenaDeColCadenas 9
insertarEnColCadenas 3 3.0 3
cadenaDeColCadenas 0
cadenaDeColCadenas 3
cadenaDeColCadenas 9
insertarEnColCadenas 2 2.0 3
insertarEnColCadenas 5 5.0 3
insertarEnColCadenas 8 8.0 3
insertarEnColCadenas 1 1.0 3
cadenaDeColCadenas 0
cadenaDeColCadenas 3
cadenaDeColCadenas 9
insertarEnColCadenas 8 8.0 0
insertarEnColCadenas 9 9.0 0
insertarEnColCadenas 5 5.0 8
insertarEnColCadenas 6 6.0 8
insertarEnColCadenas 7 7.0 8
insertarEnColCadenas 4 4.0 9
cadenaDeColCadenas 0
cadenaDeColCadenas 1
cadenaDeColCadenas 3
cadenaDeColCadenas 8
cadenaDeColCadenas 9
Fin