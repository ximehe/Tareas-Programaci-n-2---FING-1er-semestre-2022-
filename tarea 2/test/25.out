1>#  implementar cadenaDeColCadenas.
2>La cadena 0 es :
3>La cadena 9 es :
4>Se insertó (3,3.00) en la cadena 3.
5>La cadena 0 es :
6>La cadena 3 es :(3,3.00)
7>La cadena 9 es :
8>Se insertó (2,2.00) en la cadena 3.
9>Se insertó (5,5.00) en la cadena 3.
10>Se insertó (8,8.00) en la cadena 3.
11>Se insertó (1,1.00) en la cadena 3.
12>La cadena 0 es :
13>La cadena 3 es :(1,1.00)(8,8.00)(5,5.00)(2,2.00)(3,3.00)
14>La cadena 9 es :
15>Se insertó (8,8.00) en la cadena 0.
16>Se insertó (9,9.00) en la cadena 0.
17>Se insertó (5,5.00) en la cadena 8.
18>Se insertó (6,6.00) en la cadena 8.
19>Se insertó (7,7.00) en la cadena 8.
20>Se insertó (4,4.00) en la cadena 9.
21>La cadena 0 es :(9,9.00)(8,8.00)
22>La cadena 1 es :
23>La cadena 3 es :(1,1.00)(8,8.00)(5,5.00)(2,2.00)(3,3.00)
24>La cadena 8 es :(7,7.00)(6,6.00)(5,5.00)
25>La cadena 9 es :(4,4.00)
26>Fin.
